site/content/entry/yuzu-progress-report-may-2021/index.md

```diff
@@ -0,0 +1,353 @@
++++
+date = "2021-06-05T12:00:00-03:00"
+title = "Progress Report May 2021"
+author = "GoldenX86"
+coauthor = "Honghoa"
+forum = 0
++++
+
+Hola yuz-ers! Here we are with May’s progress report, with dozens of additions to discuss.
+This month we include kernel changes, input and UI improvements, graphical fixes, the saga of the legendary *dot*, and even free extra VRAM!
+
+<!--more-->
+
+## Pokémon Snap, but it’s New
+
+`New Pokémon Snap`’s release left us with tons of work to do to make the game playable. To begin with, Snap experiences crashes during gameplay, 
+an issue [epicboy](https://github.com/ameerj) was not happy about. 
+Now, yuzu has an optimized fast path for `uniform bindings`, the conditions to take advantage of it are that buffers must be both small and *non-null*. Turns out, 
+null buffers managed to skip this check, causing stabilities along the way.
+[Properly checking for those zero sized buffers](https://github.com/yuzu-emu/yuzu/pull/6322) fixed the stability issues the new Snap was facing.
+
+{{< single-title-imgs
+    "New Pokémon Snap"
+    "./1.png"
+    "./2.png"
+    "./3.png"
+  >}}
+
+[Morph](https://github.com/Morph1984) has been working hard on implementing much needed fixes in the file system emulation of yuzu.
+In this particular case, [improving the accuracy of CreateFile](https://github.com/yuzu-emu/yuzu/pull/6265) fixes the saving and loading issues Snap was experiencing.
+Turns out, the error codes of conditions like “parent directory doesn’t exist” or “path already exists” were incorrect, causing havoc in the file system emulation.
+
+Snap also revealed flaws in our touch screen gestures emulation. 
+Thanks to work done by [german77](https://github.com/yuzu-emu/yuzu/pull/6275), yuzu [now properly supports multiple fingers](https://github.com/yuzu-emu/yuzu/pull/6267).
+
+GPU emulation related issues stop this game from being perfectly playable. In its current status, photos are not recognized at the evaluation.
+This can be slightly improved by using High GPU accuracy in OpenGL with Assembly Shaders enabled, or for non-Nvidia users, using Extreme GPU accuracy on Vulkan.
```

Comment:
Hmm, I'll try to reword it.

Reply:
GPU emulation related issues stop this game from being perfectly playable. In its current status, photos are not recognized when they are evaluated at the end of a stage.
This can be partially solved by using High GPU accuracy in OpenGL with Assembly Shaders enabled, or for non-Nvidia users, using Extreme GPU accuracy on Vulkan.

Sounds better?